src/Hit/Git/Amend.hs

```diff
@@ -0,0 +1,17 @@
+-- | Everything related to the `hit amend` command
+
+module Hit.Git.Amend
+    ( runAmend
+    ) where
+
+import System.Process (callCommand)
+
+import Hit.Core (PushBool (..))
+import Hit.Git.Push (runPush)
+
+-- | @hit amend@ command.
+runAmend :: Bool -> IO ()
+runAmend localAmend = do
+    callCommand "git add ."
+    callCommand "git commit --amend --no-edit"
```

Comment:
We're using `shellmet` library to call shell commands from Haskell. To make it work, you need to import orphan instance from `Shellmet`:

* https://github.com/kowainik/shellmet

You probably can put this instance into custom `Prelude` in this project

Reply:
Thanks, that's why it didn't work as before :)

Reply:
I removed the `callCommand` calls and recoverd the Shellmet behaviour.
I decided against putting it into custom `Prelude` because at first, I don't know how to do it and second I don't think that **any** module should have access to it.

Reply:
I removed the callCommand calls and recoverd the Shellmet behaviour.
I decided against putting it into custom Prelude because at first, I don't know how to do it and second I don't think that any module should have access to it.